salt/example.top

```diff
@@ -8,5 +8,6 @@ base:
         - elife.aws-cli
         - elife.external-volume
         - elife.external-volume-srv
+        # - elife.newrelic-python
```

Comment:
locally this won't work, as there is no license being provided; so it is always correct to comment it out in example.top